### PR TITLE
Stop Using apollo-boost for Setting Flexibility

### PR DESCRIPTION
**Pull request by hitochan777**
`apollo-boost` allows you to create apollo client instance easily, because it comes with all the basic libraries for http link, caching, etc.
However, it
* lacks customization flexibility
* only supports absolute path to specify GraphQL endpoint

For these reasons, I replaced `apollo-boost` with its underlying libraries.

### CODE REVIEW

File: logger.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import createPino from "pino";
```
Comment (hitochan777) [inline logger.js:1]: Since eslint complains if you use `console.log`, here I set up logger with pino (which is a lightweight logging library).

**Comment by picklu** [comment]
@hitochan777 If `apollo-boost` is that useful then you may keep it and use a port number from an environment variable so that the apollo-client and the express server use the same port. There is a very handy library for cross-platform environment variables called `dotenv`.

**Comment by hitochan777** [comment]
@picklu Port number is the only one problem you had?

**Review by picklu — COMMENTED** [review]
Is it possible to write the uri of the apollo-client as uri:  `http://localhost:${PORT}/graphql` ? If possible then you may keep using apollo-boost with a port number from an environment variable so that the apollo-client and the express server use the same port for development environment. There is a very handy library for cross-platform environment variables called `dotenv`.

**Comment by hitochan777** [comment]
https://github.com/chingu-voyages/v8-bears-team-06/blob/70cb4d597907679d1bba78a1573d94f2d6413ee0/server/index.js#L11

If you want to use custom port, it's better to export and reuse the port above.
However, if we assume that the GraphQL server domain is the same as the web server (express), (which is true in our case), then it's much simpler to use relative path.

**Comment by picklu** [comment]
Yes, in our case, the relative path for graphql is a better option.

**Review by willjw3 — APPROVED** [review]
I'm ok with not using it if you think it's better @hitochan777.

**Comment by hitochan777** [comment]
Since nobody seems to have any objection on this, i will merge this.

**Pull request merged**